src/adminPartMain/kotlin/TestsAssociatedWithBuildStorage.kt

```diff
@@ -0,0 +1,35 @@
+package com.epam.drill.plugins.coverage
+
+import java.util.concurrent.ConcurrentHashMap
+
+object TestsAssociatedWithBuildStorageImpl : TestsAssociatedWithBuildStorage {
```

Comment:
if I were you, would rather specify in implementation's name, which storage is used. for example MutableMapTestsAssociatedWithBuildStorage. still the interface's name is terrifying long.